### DISCUSSION TITLE
Specify Spotify install location

### DISCUSSION THREAD

**Discussion by NotSpaulding** [SpotX-Official/SpotX#461] [Question]
I am using [Soggfy](https://github.com/Rafiuth/Soggfy) which extracts Spotify to a non-ordinary location.

If I recall correctly you used to be able to do this using a flag on the installer but I can't seem to find that flag now.

Is this still possible to do? How can I inject SpotX to an arbitrary path Spotify is installed in?

**Comment by amd64fox** [comment]
no, there was no such flag in SpotX, use [manual installation](https://github.com/Rafiuth/Soggfy#manual-install)

Just in case, I condemn this method of downloading music. You risk getting your account banned.

**Reply by NotSpaulding** [reply]
Hm, I thought I had injected SpotX into Soggfy's directory sometime in the past but I guess not. Thanks for your help and the quick response!
